nixos/tests/beanstalkd.nix

```diff
@@ -2,22 +2,22 @@ import ./make-test.nix ({ pkgs, lib, ... }:
 
 let
   produce = pkgs.writeScript "produce.py" ''
-    #!${pkgs.python2.withPackages (p: [p.beanstalkc])}/bin/python
+    #!${pkgs.python3.withPackages (p: [p.beanstalkc])}/bin/python
     import beanstalkc
 
     queue = beanstalkc.Connection(host='localhost', port=11300, parse_yaml=False);
-    queue.put('this is a job')
-    queue.put('this is another job')
+    queue.put(b'this is a job')
+    queue.put(b'this is another job')
   '';
 
   consume = pkgs.writeScript "consume.py" ''
-    #!${pkgs.python2.withPackages (p: [p.beanstalkc])}/bin/python
+    #!${pkgs.python3.withPackages (p: [p.beanstalkc])}/bin/python
```

Comment:
same here. I suggest you move the `pkgs.python2.withPackages (p: [p.beanstalkc])` as `pythonEnv` to a `let .. in`

Reply:
Sounds good.